packages/firebase_auth/firebase_auth_platform_interface/CHANGELOG.md

```diff
@@ -1,20 +1,23 @@
+## 1.1.4
+
+* Fixed typo on private method name.
+
 ## 1.1.3
 
-- Added support for `confirmPasswordReset`
+* Added support for `confirmPasswordReset`
```

Comment:
I think we should probably try not to change the bullet types

Reply:
I've seen this done in other plugins, was just following the trend. Will revert.